src/drawer/components/MoreProductSection/MoreProductSection.tsx

```diff
@@ -10,26 +17,34 @@ export const MoreProductSection = ({
   providerName: string;
   providerId: string;
 }) => {
-  const { isSuccess, data } = useGetProductByProvider({ providerId });
+  const { data } = useGetProductByProvider({ providerId });
+
+  const navigate = useNavigate();
 
+  if (data.length === 0) return;
   return (
```

Comment:
코어 자바스크립트라는 책에서 `undefined`는 '값을 대입하지 않은 변수에 접근하고자 할 때 자바스크립트 엔진이 반환해주는 값’으로만 존재할 수 있도록 값이 없을을 명시적으로 나타내고 싶을 때는 `null`을 사용하라 하던데 `return null`을 해주는게 어떨까요

```suggestion
  if (data.length === 0) return null;
```

Reply:
헉 코멘트가 많아서 못 보고 있었어요 반영했습니다~~~!